### PR TITLE
[PKG-2497] mizani 0.9.2

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Changelog: https://github.com/has2k1/mizani/releases
License: https://github.com/has2k1/mizani/blob/v0.9.2/LICENSE
Requirements: https://github.com/has2k1/mizani/blob/v0.9.2/pyproject.toml

Actions:
1. Clean up the recipe:
- Remove unused jinja2 variables, just KISS
- Remove `noarch` python
- Skip `py<38`
- Add flags `--no-deps --no-build-isolation` to `script`
- Add missing `wheel` to `host`
- Fix `python` in `host` and `run`
2. Add selectors for `backports.zoneinfo`
3. Do not use the `[win]` selector for `tzdata` as it's not built for this platform however the compiled package can be used on `Windows` https://github.com/AnacondaRecipes/tzdata-feedstock/blob/07c7f5de102ca5bf9e793838170f26fadd0c3a6b/recipe/meta.yaml#L19
4. Fix `source_files`
5. Remove debug commands from `test/requires` and fix the `pytest` command
6. Update `summary`
7. Add `description`

Notes:
- `plotnine 0.12.1` requires `mizani >=0.9.0`, see https://github.com/has2k1/plotnine/blob/v0.12.1/pyproject.toml#L27

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,65 +1,59 @@
...
+    - 'mizani/tests/*.py'
   commands:
-    # The pip version can reported incorrectly if setuptools_scm isn't available
-    # at build time
-    - pip list | grep -i {{ name }} | grep {{ version }}
-    - pip check
-    - echo $PWD
-    - ls -larth
-    - pytest
+    - pip check  # [not win]
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:46]: skipping the check on windows, does it mean this package is still usable for windows ?

Comment (skupr-anaconda) [reply recipe/meta.yaml:46]: Yes

**Review by boldorider4 — APPROVED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,65 +1,59 @@
...
     - setuptools_scm
-
+    - wheel
   run:
-    - python >=3.8.0
-    - numpy >=1.19.0
-    - scipy >=1.5.0
-    - pandas >=1.3.5
+    - python
+    - {{ pin_compatible('numpy') }}
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - numpy>=1.19.0
```
pin_compatible is suitable when numpy is set in host.
mizani only uses the python api of numpy, so we can go with what is defined in pyproject.toml

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,65 +1,59 @@
...
+    - backports.zoneinfo  # [py<39]
     - matplotlib-base >=3.5.0
-    - backports.zoneinfo  # py<3.9 conda does not properly support environment markers
-    - tzdata  # os == windows
+    - pandas >=1.3.5
+    - scipy >=1.5.0
+    - tzdata
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - tzdata  # [win]
```
https://github.com/has2k1/mizani/blob/v0.9.2/pyproject.toml#L27

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,65 +1,59 @@
...
 
 test:
-  requires:
-    - pip
-    - pytest
-    - pytest-cov
   imports:
     - mizani
     - mizani.external
     - mizani.tests
```
Comment (cbouss) [inline recipe/meta.yaml:37]:
```suggestion
    - mizani.tests
    - mizani.colors
```

File: recipe/meta.yaml [hunk 5]
```diff
@@ -1,65 +1,59 @@
...
+  requires:
+    - pip
+    - pytest
+    - pytest-cov
```
Comment (cbouss) [inline recipe/meta.yaml:42]:
```suggestion
```
No need to include it if it is not used.

Comment (skupr-anaconda) [reply recipe/meta.yaml:42]: No, in this case, tests will fail without` pytest-cov`. I tried but the maintainers hardcoded this inside their tests

**Review by cbouss — APPROVED** [review]

**Pull request merged**